gem/forms.py

```diff
@@ -0,0 +1,50 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.utils.translation import ugettext_lazy as _
+from gem.constants import GENDERS
+
+
+class RegistrationForm(forms.Form):
```

Comment:
@KaitCrawford I think you can just inherit from `molo.profiles.forms` - I noted it for TuneMe after the fact. In which case you only need to specify the Gender field.